src/moi_wrapper.jl

```diff
@@ -50,6 +50,19 @@ function diff_optimizer(
         with_bridge_type,
         with_cache_type,
     )
+    if !isnothing(with_bridge_type)
+        # removed because of the `ZerosBridge` issue:
+        # https://github.com/jump-dev/MathOptInterface.jl/issues/2861
+        # - zeros bridge does not support duals because it cumbersome
+        # - many bridges do not support get ConstraintFunction because it is cumbersome
```

Comment:
Why would we need ConstraintFunction ? I don't understand this one

Reply:
This is something needed by the tests but in practice, no one asks for ConstraintFunction to the bridges outside of the tests. It's not clear in your comment if DiffOpt needs to query ConstraintFunction or not